charts/kusion/README.md.gotmpl

```diff
@@ -79,8 +85,8 @@ helm install kusion-release kusionstack/kusion \
```

Comment:
The base64 encoding command has been modified to use 'cat' piped to 'base64' instead of direct 'base64 -w 0'. This change improves cross-platform compatibility but should be tested on different operating systems to ensure consistent behavior.